datalad/install.py

```diff
@@ -318,12 +312,37 @@ def _install_via_conda(self, version, tmpdir):
             check=True,
         )
         if self.annex_bin != conda_bin:
-            os.makedirs(self.annex_bin, exist_ok=True)
-            with dirchanged(self.annex_bin):
-                for fname in glob("../miniconda/bin/git-annex*"):
-                    os.symlink(fname, os.path.basename(fname))
+            annex_bin = Path(self.annex_bin)
+            annex_bin.mkdir(parents=True, exist_ok=True)
+            for p in Path(conda_bin).glob("git-annex*"):
+                (annex_bin / p.name).symlink_to(p.resolve())
         self.post_install()
 
+    def install_miniconda(self, miniconda_path, batch=False):
+        miniconda_script = "Miniconda3-latest-Linux-x86_64.sh"
```

Comment:
while at it, could you make it also at least for OSX for which there is also a https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh which I hope behaves identically to the one for linux?

Ideally should have one for windows, but since it is a python script, it is somewhat of a chicken/egg issue so we could postpone (evil plan: I want later to try to build some executable version of this self-contained script)

Reply:
Done.